README.md

```diff
@@ -21,16 +22,22 @@ When the widget is configured as above, it sends postMessages via navigation eve
 
 In the demo app, you will want to use `appscheme` as the `ui_message_webview_url_scheme`.
 
-It is *imperative* that your native application intercept *all* navigation events. In addition to the widget events, the widget also has links to bank and/or financial institution sites. You will want to intercept these and handle them accordingly. Failure to do so may result in your WebView being replaced by the link or URL event.
+It is *imperative* that your native application intercept *all* navigation events. In addition to the widget events, the widget also has links to bank and/or financial institution sites. You will want to intercept these and send them to the user agent isntead of overriding the webview. Failure to do so may result in your WebView being replaced by the link or URL event.
```

Comment:
```suggestion
It is *imperative* that your native application intercept *all* navigation events. In addition to the widget events, the widget also has links to bank and/or financial institution sites. You need to intercept these and send them to the user agent instead of overriding the WebView. Failure to do so may result in your WebView being replaced by the link or URL event.
```